src/App.js

```diff
@@ -1,14 +1,24 @@
 import React, { Component } from 'react';
-import './App.css';
-import Calculator from './components/Calculator';
+import { Route, Routes, BrowserRouter } from 'react-router-dom';
+import Math from './pages/calculatorPage';
+import Quote from './pages/quotePage';
+import Home from './pages/homePage';
+import NavBar from './components/navBar/navBar';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please kindly fix your linters errors instead of disabling them. Fix linters errors help us to have clean and professional codebase 🙃 😉